test/go.sum

```diff
@@ -0,0 +1,677 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Is this relevant if the tests will run in an ephemeral environment? if not, we should gitignore it

Reply:
Yes.. the docker image has only the base GO dependencies. For now, each TF module will be having its own dependencies and will download them in every run.